src/js/components/productSupplier/create/subsections/FixedPrice.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { Controller } from 'react-hook-form';
+
+import CheckBox from 'components/form-elements/v2/Checkbox';
+import DateField from 'components/form-elements/v2/DateField';
+import TextInput from 'components/form-elements/v2/TextInput';
+import Subsection from 'components/Layout/v2/Subsection';
+
+const FixedPrice = ({ control, errors }) => (
+  <Subsection
+    title={{
+      label: 'react.productSupplier.form.subsection.fixedPrice',
+      defaultMessage: 'Fixed Price',
+    }}
+    collapsable={false}
+  >
+    <div className="form-grid-3">
+      <Controller
+        name="contractPriceEach"
+        control={control}
+        render={({ field }) => (
+          <TextInput
+            {...field}
+            decimal={2}
+            type="number"
+            errorMessage={errors.contractPriceEach?.message}
+            title={{
+              id: 'react.productSupplier.form.contractPriceEach.title',
+              defaultMessage: 'Contract Price Each',
+            }}
+            tooltip={{
+              id: 'react.productSupplier.form.contractPriceEach.tooltip',
+              defaultMessage: 'Fixed price per unit guaranteed by a contract with the supplier',
+            }}
+          />
+        )}
+      />
+      <Controller
+        name="priceValidUntil"
```

Comment:
let's name it as `contractPriceValidUntil`, as it is pointing to the `contractPrice` of `ProductSupplier`.